ihatemoney/migrations/versions/52903c4b2459_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 52903c4b2459
+Revises: 6c6fb2b7f229
+Create Date: 2019-10-20 20:31:49.848058
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "52903c4b2459"
+down_revision = "6c6fb2b7f229"
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("bill", sa.Column("type", sa.UnicodeText(), nullable=True))
```

Comment:
I don't think we want a nullable there but more a loan by default.

Reply:
When I look at all the other migration scripts every single new column is nullable except "id". I guess a default value in the model definition should be enough.